src/platform-includes/getting-started-install/javascript.astro.mdx

```diff
@@ -0,0 +1,9 @@
+We recommend installing the SDK by using the `astro` CLI:
+
+```bash
+npx astro add @sentry/astro
+```
```

Comment:
l: Wondering, is there a chance that astro will change _some_ API and this will break? If so, will it make sense to build and recommend a wizard of some sort that will work independently of the astro CLI? (no action required right now, just sharing my thoughts)

Reply:
Afaict, this is pretty stable for now (but the current approach has its limitations). We should keep an eye open for if they change things (I need to finally write up that RFC for Astro I've mentioned some time ago). 

We could build a wizard because right now, folks still need to manually add their DSN, source maps config, etc. Ideally, the Astro CLI would invoke it for us, so we could retain the "Astro native" approach. But I guess that's just wishful thinking at this point 😅